dashboard/app/controllers/pd/workshop_survey_controller.rb

```diff
@@ -10,11 +10,8 @@ def new
 
     workshop = @pd_enrollment.workshop
 
-    user = @pd_enrollment.user
-    user_surveys = Pd::WorkshopSurvey.
-      joins(:pd_enrollment).
-      where(pd_enrollments: {user_id: user.id})
-    is_first_survey = user_surveys.count == 0
+    user_surveys = Pd::WorkshopSurvey.find_by_user(@pd_enrollment.user)
```

Comment:
oh, I should have been more explicit; this PR depends on https://github.com/code-dot-org/code-dot-org/pull/15285, which adds that method: https://github.com/code-dot-org/code-dot-org/pull/15285/files#diff-5e76c0a573338895ba5b0620f20c93e7R86